### DISCUSSION TITLE
A possible solution to the IMAGE display error

### DISCUSSION THREAD

**Discussion by ghost** [dompdf/dompdf#2516] [General]
Young people, I am new to using this library and I found the problem that I cannot view images that I was looking for and I cannot find a solution, I have the latest version of the library and PHP 8 I cannot find a solution, currently I have had to translate information to see if I achieve something , doing some tests on the images I found a possible quick solution and it is the following one MUST COPY THE IMAGES YOU NEED IN THE ROOT OF THE DOMPDF LIBRARY and that would be your path BUT THE TRICK IS THAT THESE IMAGES MUST GO IN SVG FORMAT and so on is how I solve.

**Comment by bsweeney** [comment]
It seems that you're trying to load images from the local file system. In that case you should update the chroot option to indicate the correct path. As for the format, Dompdf should support SVG and JPG out of the box. PNG is supported if the GD library is installed.
